### PR TITLE
Add disk labels

### PR DESCRIPTION
**Pull request by KaiWaldrant**
## Describe your changes

Add labels to increase storage if necessary

## Issue ticket number and link
Closes #xxxx (Replace xxxx with the GitHub issue number)

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [x] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [ ] Proposed changes are described in the CHANGELOG.md

- [x] CI Tests succeed and look good!

## Requirements after merging

- [ ] Need to regenerate `common/` resources

- [ ] Need to regenerate task-specific resources. Specify: <all or ...>

### CODE REVIEW

File: src/wf_utils/labels_tw.config [hunk 1]
```diff
@@ -14,6 +17,11 @@ process {
   withLabel: lowmem { memory = { get_memory( 20.GB * task.attempt ) } }
   withLabel: midmem { memory = { get_memory( 50.GB * task.attempt ) } }
   withLabel: highmem { memory = { get_memory( 100.GB * task.attempt ) } }
+
+  withLabel: lowdisk { disk = 100.GB }
+  withLabel: middisk { disk = 250.GB }
+  withLabel: highdisk { disk = 500.GB }
```
Comment (rcannood) [inline src/wf_utils/labels_tw.config:23]: Should disk also scale with attempt #? :thinking: 

E.g. attempt 1: 50GB, attempt 2: 200 GB, attempt 3: 500 GB?

Comment (KaiWaldrant) [reply src/wf_utils/labels_tw.config:23]: was thinking the same thing

Comment (KaiWaldrant) [reply src/wf_utils/labels_tw.config:23]: The issue at hand is that these processes fail with error code 1 so no real way of detecting that the disk storage is the problem.

**Review by rcannood — APPROVED** [review]

**Review by rcannood — APPROVED** [review]

File: src/wf_utils/labels_tw.config [hunk 2]
```diff
@@ -11,9 +14,19 @@ process {
...
+                      disk = { 50.GB * task.attempt } 
+                    }
+  withLabel: midmem { 
+                      memory = { get_memory( 50.GB * task.attempt ) }
+                      disk = { 100.GB * task.attempt } 
+                    }
+  withLabel: highmem {
+                      memory = { get_memory( 100.GB * task.attempt ) }
+                      disk = { 200.GB * task.attempt } 
+                    }
```
Comment (rcannood) [inline src/wf_utils/labels_tw.config:28]: Undo weird indentation ;)

```suggestion
  withLabel: lowmem { 
    memory = { get_memory( 20.GB * task.attempt ) } 
    disk = { 50.GB * task.attempt } 
  }
  withLabel: midmem { 
    memory = { get_memory( 50.GB * task.attempt ) }
    disk = { 100.GB * task.attempt } 
  }
  withLabel: highmem {
    memory = { get_memory( 100.GB * task.attempt ) }
    disk = { 200.GB * task.attempt } 
  }
```

**Review by rcannood — APPROVED** [review]

**Pull request merged**